### ISSUE TITLE
Style the "The update process was aborted prematurely" message as a proper error in update.php

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5392]
Before:
![image](https://user-images.githubusercontent.com/2423362/145712769-89fc13ea-3616-4bf5-88a3-7b0d21d89bc0.png)

After:
![image](https://user-images.githubusercontent.com/2423362/145712843-a280c6f5-b919-4b54-b288-accfa2f6fce3.png)

**Comment by klonos** [comment]
The message adds `$module . '.module`, but update hooks are actually locates in `.install` files - not in `.module` files. Should we also fix that here?

**Comment by indigoxela** [comment]
```
The update process was aborted prematurely while running <strong>update #' . $version . ' in ' . $module . '.module</strong>.
```
Looking at the current text, the intention seems to be to mention the failing module. A file extension (.module or .install) doesn't bring much benefit. Agreed, if we use an extension, then it should rather be ".install".

I think, it's OK to also fix that minimal wording glitch here. An extra issue for that seems overdone.

What seems more important to me, is to actually display the update number and module name. @klonos your screenshots don't contain this important info - is this because of your testing setup, or does that actually happen in real failures? (Never ran into this failure.)

**Comment by indigoxela** [comment]
To test this locally, I created a dummy module with dummy update hook, which only does a `die()`.

First of all: yes the required info about the module is there :+1:.

This is what we have now:

![update-error-message](https://user-images.githubusercontent.com/842066/145784806-b5630736-29a3-4838-a04d-df957aa64693.png)

TBH, I find that double mention of logging a bit too wordy:

```
The update process was aborted prematurely while running update #1001 in testthing.module. All errors have been logged. You may need to check the watchdog database table manually.
```
Why mentioning dblog twice? (In this case (with a die()) there's nothing in dblog.) I'm aware that this is nothing this PR introduces, it has been there before. There may be a reason for this, though.

**Comment by BWPanda** [comment]
It makes sense to me.
- 'Errors logged' tells where things have been saved, since the errors aren't shown here. That's helpful.
- 'Check `watchdog` manually' tells me where to look in case the errors cause the site not to work properly (i.e. in case the above link doesn't work). Also helpful.

Checked code, looks good to me.

**Comment by quicksketch** [comment]
Thanks folks! Looks good to me! I merged https://github.com/backdrop/backdrop/pull/3858 into 1.x and 1.22.x.
